terraform/modules/forest_node/user-data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Create a login user
```

Comment:
```suggestion
: create a user
# --disabled-password: the only way a user can log in is with e.g ssh
# --gecos "": don't set address and phone number
```

Print out the description as the command runs - very useful for debugging remotely failing scripts ;)

Use a comment to save information for future you/other readers.

If a reasonable reader needs to ask "why have they done it like this", it's a good place for a comment


Why have you manually set `--home` to the default value - please comment the script or remove